mapcomposer/app/static/externals/gxp/src/script/plugins/WFSSearchBox.js

```diff
@@ -124,8 +129,13 @@ Ext.namespace("gxp.plugins");
     /** api: method[addOutput]
      */
     addOutput: function(config) {
-        
-    	var controls = (!this.markerFadeoutEnable && !this.noButton) ? [ this.removeMarkerBtn, this.combo] : [this.combo];
+    	var controls;
+    	if(this.itemsContainer && (!this.markerFadeoutEnable && !this.noButton) )
+    		controls = [ new Ext.form.CompositeField( { 
+    			//layout:'hbox',
+    			items:[this.removeMarkerBtn, this.combo]})];	
```

Comment:
In the sample configuration the marker disappears with the timer ( with the 3rd search box) while on the server it doesn't disappear. Be sure it is not a bug and align the sample configuration if needed.